cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/10-max-debt-guideline.html

```diff
@@ -21,37 +21,48 @@ <h2>Could this be too much debt?</h2>
     </div>
+    <div class="debt-guideline_salary-info">
+    {{ number_callout.render({
+                'header': 'Median salary of this <span class="undergrad-content">school\'s recent students</span><span class="graduate-content">program\'s graduates</span>' | safe,
+                'data_attribute': 'data-financial-item="salary_annual"',
+                'additional_classes': 'number-callout__large',
+                'value': '$0'
+            })
+        }}
+        
 
-    <div data-state-based-visibility="debt-rule-okay">
-        {{ notification.render(
-            'default',
-            true,
-            'Your projected total debt is not higher than the <span data-state-based-visibility="no-program-selected">median salary of this school\'s recent students</span><span data-state-based-visibility="program-is-selected">median first-year salary of this program\'s graduates</span>'  | safe,
-            'Your actual salary will depend on factors like your field of work and where you live. <a href="https://www.bls.gov/bls/blswage.htm" target="_blank" class="external-link" rel="noreferrer noopener">Visit the BLS Wage Data site for more information.</a>' | safe
-        ) }}        
+        <p data-state-based-visibility="undergrad-content">Median earnings of federally-aided students working and not enrolled 6 years after starting at this school—includes students who did not graduate</p>
+        <p data-state-based-visibility="program-is-selected">Median first-year earnings of federally-aided program graduates who were working and not enrolled during the first year following graduation</p>
+
+        <h4>Your projected debt vs. the median <span class="undergrad-content">salary of recent students</span><span class="program-is-selected">first-year salary of program graduates</span></h4>
+        
+        <div id="max-debt-guideline_chart"></div>
+
+        <div data-state-based-visibility="debt-rule-violation">
+            {{ notification.render(
+                'caution',
+                true,
+                'Your projected total debt is more than the <span data-state-based-visibility="no-program-selected">median salary of this school\'s recent students</span><span data-state-based-visibility="program-is-selected">median first-year salary of this program\'s graduates</span>. Consider whether this plan is right for you.' | safe,
+                'Your projected total debt is <span data-financial-item="other_debtGuideDifference">$XXXX</span> higher than the <span data-state-based-visibility="no-program-selected">median salary of this school\'s recent students</span><span data-state-based-visibility="program-is-selected">median first-year salary of this program\'s graduates</span>. Your actual salary will depend on factors like your field of work and where you live. <a href="https://www.bls.gov/bls/blswage.htm" target="_blank" class="external-link" rel="noreferrer noopener">Visit the BLS Wage Data site for more information.</a>' | safe
```

Comment:
It looks like neither of the `no-program-is-selected` / `program-is-selected` options shows up for undergrad:

<img width="607" alt="Screen Shot 2020-07-14 at 2 46 20 PM" src="https://user-images.githubusercontent.com/778171/87480240-c0bac300-c5e1-11ea-8009-9287ac6ddfa9.png">

Reply:
Apologies. I've changed many instances of visibility based on program selection to visibility based on program level because the content is mutually exclusive between grads and undergrads. Some of it didn't get done correctly, but I think I've fixed them now.